html/browsers/browsing-the-web/navigating-across-documents/failure-check-sequence.https.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Sequence of the checks performed against a navigation response</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+'use strict';
+const collect = (win) => {
+  const report = new Promise((resolve) => {
+    if (!win.ReportingObserver) {
+      return;
+    }
+
+    const observer = new win.ReportingObserver(resolve);
+    observer.observe();
+  }).then((reports) => reports[0].toJSON().type);
+  // Although CSP also makes use of ReportingObserver, monitoring this event
+  // allows the test to provide value to implementations that have not yet
+  // integrated CSP and Reporting.
```

Comment:
Are there any modern implementations of that sort?

Reply:
Both Firefox 81 and Safari 13.1 emit the `securitypolicyviolation` event, but neither issue a `csp-violation` Report. Would you like me to mention that in the comment?

Reply:
Something like "(as of the time of this writing, Firefox and Safari)" would be helpful I think, but I don't feel strongly.

Reply:
You got it